src/libwalletqt/UnsignedTransaction.h

```diff
@@ -26,13 +26,6 @@ class UnsignedTransaction : public QObject
     };
     Q_ENUM(Status)
 
-    enum Priority {
-        Priority_Low    = Monero::UnsignedTransaction::Priority_Low,
-        Priority_Medium = Monero::UnsignedTransaction::Priority_Medium,
-        Priority_High   = Monero::UnsignedTransaction::Priority_High
-    };
-    Q_ENUM(Priority)
-
```

Comment:
As commented here (https://github.com/monero-project/monero/commit/a9cae0abe7b71c4a112089016cb16ee1ee12d69f#commitcomment-27179654), the same enum defined in https://github.com/monero-project/monero-gui/blob/master/src/libwalletqt/PendingTransaction.h seems also unused. Wouldn't it then make sense to also remove that, too?

Reply:
My overlooking again: `PendingTransaction::Priority` **is** used in `libwalletqt/Wallet.*`, so it can't be removed.